test/core/api/transport/jrpc_handle_batch_test.cpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
+#include <jsonrpc-lean/server.h>
+
+#include "api/jrpc/jrpc_handle_batch.hpp"
+
+using kagome::api::JrpcHandleBatch;
+
+struct JrpcHanldeBatchTest : ::testing::Test {
+  jsonrpc::Server jsonrpc_handler_;
+  jsonrpc::JsonFormatHandler format_handler_;
+  void SetUp() override {
+    jsonrpc_handler_.RegisterFormatHandler(format_handler_);
+  }
+};
+
+TEST_F(JrpcHanldeBatchTest, Test) {
```

Comment:
Please comment. And it looks like it'd be better to split it in several short tests.